src/sentry/static/sentry/app/components/forms/selectField.jsx

```diff
@@ -40,6 +49,10 @@ export default class SelectField extends FormField {
         value={this.state.value}
         onChange={this.onChange.bind(this)}
         clearable={this.props.clearable}
+        // TODO(dcramer): if we're using multiple as our top level API, we
```

Comment:
do you have thoughts on this billy? i'm happy to go with whatever yall decide, but to reiterate my opinions:

- the ``<select>`` html element uses multiple over multi
- we shouldn't couple our APIs to react-select (related but unrelated here)
- we can't change this easily in the future, and historically ``multiple`` would have worked but I couldn't see us using it anywhere

Reply:
Yeah I think `<select>`using `multiple` is a good enough reason. We should move this logic into `SelectControl` though

Reply:
done and done